content/post/051-new-ipfs-release-process.md

```diff
@@ -0,0 +1,229 @@
+We've made three specific changes to the release process:
+
+1. To address the stability issues, we've introduced a new release process that
+   involves extensively testing releases in a wide variety of production
+   environments.
+2. To address the issue of slow releases, we've introduced a 6 week release
+   cycle.
+3. To address the issue of slow bug fixes, we've switched to semver and
+   introduced patch releases. The first patch release will be 0.4.22 and the
+   next _feature_ release will be 0.5.0.
+   
+### Early Testers Program
+
+The early testers program allows groups using go-ipfs in production to
+self-volunteer to help with testing `go-ipfs` release candidates in development
+and production environments. While we invite the _entire_ community to help test
+releases, members of the early testers program participate directly and actively
+in every release.
+
+Early testers will deploy release candidates to dev and prod environments,
+giving us quick feedback on any regressions or performance changes they notice.
+This means we can get some quick feedback from heavy users before we cut the
+official release.
+
+This program currently includes:
+
+- Infura
+- Textile
+- Pinata
+- RTrade
+- QRI
+- Siderus
+
+### Cycle
+
+Baring any feature freezes, go-ipfs will now cut a new release approximately
+every 6 weeks. Specifically, we will aim to branch off a new release every 6 weeks and
+then run through the release process in an expected 3 weeks.
+
+If the release process runs under or over the expected 3 weeks, the next release
+will aim to branch at the 6 week mark, regardless. That way, even if we don't
+release on schedule, we can still maintain a 6 week release cadence.
+
+#### Patch Releases
+
+Given the increased structure and extensive testing in this release process, we need a way
+to quickly release fixes for critical regressions, should they arise. If we fix
+a critical regression in a go-ipfs release, we will create a _patch_ release for
+this regression based on the current stable release.
+
+This patch release will still undergo an abbreviated release testing process, but we expect
+it to take 2-3 days, rather than weeks:
+
+1. Less than a day for internal testing.
+2. 1-2 days for production testing by members of the early testers program.
+
+Note: This release process _does not_ introduce long term support releases.
+Patches will only be applied to the latest release and will not be backported.
+Furthermore, the next feature release will likely include additional bug fixes not deemed critical to fix in
+a patch release.
+
+#### Semver
+
+This release process finally switches go-ipfs over to semver. Like many pre-1.0
+project, go-ipfs has reserved MINOR releases for large breaking changes.
+However, this means we can't distinguish between true patch releases (bug fixes
+applied to the previous release) and feature releases (minor releases).
+
+This means:
+
+* Minor releases will no longer signal large breaking changes.
```

Comment:
```suggestion
* Minor releases will signal large breaking changes.
* Patch releases will no longer signal large breaking changes.
```

Reply:
This section was correct. We were using minor releases to indicate large changes that often required network upgrades.

Reply:
Ah gotcha - I’ve never seen a minor release change so I assumed it was a typo!

Reply:
"no longer signal large breaking changes, but just regular new features and protocol improvements"?

Reply:
Hm. I guess that may have only been 0.4.0? I've clarified this section a bit.